data/json/monsters/nether.json

```diff
@@ -871,6 +896,31 @@
       }
     ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_STAR_VAMPIRE_COILING_DRINK_BLOOD",
```

Comment:
Do these EOC need to check for Bleed Immune or is that happening somewhere else already so it's not visible in this PR?

Reply:
Those are checked on the attack conditions to determine if it tries to drink blood or just bite: https://github.com/CleverRaven/Cataclysm-DDA/blob/06c6e3fad75c29494cccac7c405186c461be7ec8/data/json/monsters/nether.json#L577